playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml

```diff
@@ -1,157 +1,146 @@
-- hosts: all
+- hosts: k8s-master
+  name: Bootstrap kubernetes master
+  become: yes
+
+  pre_tasks:
+    - name: create /etc/kubernetes
+      file:
+        path: /etc/kubernetes
+        state: directory
+
+    - name: ship kubeadm.yaml
+      copy:
+        dest: /etc/kubernetes/kubeadm.yaml
+        content: |
+          apiVersion: kubeadm.k8s.io/v1beta2
+          kind: ClusterConfiguration
+          networking:
+            podSubnet: 192.168.0.0/16
+          kubernetesVersion: {{ kubernetes_version | default('latest') }}
+          apiServer:
+            extraArgs:
+              "cloud-provider": "external"
+          controllerManager:
+            extraArgs:
+              "cloud-provider": "external"
+          ---
+          apiVersion: kubeadm.k8s.io/v1beta2
+          kind: InitConfiguration
+          nodeRegistration:
+            kubeletExtraArgs:
+              cloud-provider: "external"
+
   roles:
```

Comment:
Quick test result:
```
ERROR! the role 'config-golang' was not found in /home/ubuntu/openlab-zuul-jobs/playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/roles:/home/ubuntu/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/home/ubuntu/openlab-zuul-jobs/playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance

The error appears to be in '/home/ubuntu/openlab-zuul-jobs/playbooks/cloud-provider-openstack-acceptance-test-e2e-conformance/run.yaml': line 34, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  roles:
    - config-golang
      ^ here
```

Reply:
Thanks for the testing, you can find the roles used in this playbook here https://github.com/theopenlab/openlab-zuul-jobs/tree/master/roles, you need to copy the role folder to your ansible role path.